bodlambdalogreadaccess_policy.tf

```
@@ -0,0 +1,37 @@
+# IAM policy documents that allow reading of CloudWatch logs related
+# to BOD 18-01 scanning.  This will be applied to the IAM group we are
+# creating.
+data "aws_iam_policy_document" "bodlambdalogreadaccess_policy_doc" {
+  # Allow listing of CloudWatch log groups.
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "logs:DescribeLogGroups",
+    ]
+
+    resources = [
+      "*",
+    ]
+  }
+
+  # Allow access to the individual log groups
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "logs:DescribeLogStreams",
+      "logs:FilterLogEvents",
+      "logs:GetLogEvents",
+    ]
+
+    resources = values(data.aws_cloudwatch_log_group.bod_lambda_logs)[*].arn
```

I personally think a list comprehension is clearer than the splat syntax:
```suggestion
    resources = [for k,v in data.aws_cloudwatch_log_group.bod_lambda_logs : v.arn]
```

---

> I personally think a list comprehension is clearer than the splat syntax

Seconded.  We have been gradually de-splatting our Terraform code whenever possible.

---

Works for me. I was on the fence between the two and only went with the splat because it was closer to what was already approved. Please see [531bb24](https://github.com/cisagov/bod-18-01-cloudwatch-users/pull/9/commits/531bb248d40269ee760ab0de2900f6629ee6d5a1) for the change.